website/docs/cli/commands/describe/describe-affected.mdx

```diff
@@ -200,20 +201,21 @@ Affected components and stacks:
+| `--repo-path`                      | Path to the already cloned target repository with which to compare the current branch.<br/>Conflicts with `--ref`, `--sha`, `--ssh-key` and `--ssh-key-password`                                                                                                                                                                                                                                             | no       |
+| `--verbose`                        | Print more detailed output when cloning and checking out the target<br/>Git repository and processing the result                                                                                                                                                                                                                                                                                             | no       |
+| `--include-spacelift-admin-stacks` | Include the Spacelift admin stack of any stack<br/>that is affected by config changes                                                                                                                                                                                                                                                                                                                        | no       |
+| `--clone-target-ref`               | Clone the target reference with which to compare the current branch.<br/>`atmos describe affected --clone-target-ref=true`<br/><br/>If set to `false` (default), the target reference will be checked out instead<br/>This requires that the target reference is already cloned by Git,<br/>and the information about it exists in the `.git` directory                                                      | no       |
+| `--include-dependents`             | Include the dependent components and stacks.<br/>`atmos describe affected --include-dependents=true`                                                                                                                                                                                                                                                                                                         | no       |
+| `--include-settings`               | Include the `settings` section for each affected component.<br/>`atmos describe affected --include-settings=true`                                                                                                                                                                                                                                                                                            | no       |
```

Comment:
Please use definition lists

Reply:
Use this format for required/optional

```
<dl>
  <dt>`--repo-path` __optional__</dt>
  <dd>
  Path to the already cloned target repository with which to compare the current branch.
Conflicts with `--ref`, `--sha`, `--ssh-key` and `--ssh-key-password`  
  </dd>
</dl>
```

Reply:
Ideally. I want to drop tables _everywhere_ that we use them like this, but let's not start new ones for now.